ci/kokoro/docker/Dockerfile.ubuntu

```diff
@@ -23,6 +23,7 @@ RUN apt-get update && \
         ccache \
         clang \
         clang-9 \
+        clang-format-9 \
```

Comment:
Ah, I see why you did this.  I still it is better to run this build in the same distro that runs the formatting.  Soon (around 2021-04, maybe 2021-10) we will update the `clang-tidy` build to the next Fedora, get a new `clang-format` with it, and there is no guarantee you will be able to find the same version in Ubuntu:18.04 (or Ubuntu:20.04).